### ISSUE TITLE
regression: Tabular Editor 2.3 crashes directly on startup (2.2 works perfectly)

### ISSUE THREAD

**Issue by fmms** [otykier/TabularEditor#45]
Hi,

former versions of tabular Editor worked perfectly (2.2.6260). With yesterdays release of 2.3 I get a crash right at startup.

```
Problem signature:
  Problem Event Name:	CLR20r3
  Problem Signature 01:	TabularEditor.exe
  Problem Signature 02:	2.3.6320.43145
  Problem Signature 03:	58fa8073
  Problem Signature 04:	mscorlib
  Problem Signature 05:	4.6.1590.0
  Problem Signature 06:	5787ed44
  Problem Signature 07:	1587
  Problem Signature 08:	fc
  Problem Signature 09:	System.IO.DirectoryNotFound
  OS Version:	6.1.7601.2.1.0.16.7
  Locale ID:	1031
  Additional Information 1:	0a9e
  Additional Information 2:	0a9e372d3b4ad19135b953a78882e789
  Additional Information 3:	0a9e
  Additional Information 4:	0a9e372d3b4ad19135b953a78882e789

Read our privacy statement online:
  http://go.microsoft.com/fwlink/?linkid=104288&clcid=0x0409

If the online privacy statement is not available, please read our privacy statement offline:
  C:\Windows\system32\en-US\erofflps.txt
```

![grafik](https://cloud.githubusercontent.com/assets/908734/25302591/8a7af374-2741-11e7-864c-5a3751e9f3dd.png)

Anything more I need to provide to debug the issue?

Regards
Felix

**Comment by otykier** [comment]
Hi Felix. I just released a new build (2.3.6321) which should solve this issue. You can get it from the [release page](https://github.com/otykier/TabularEditor/releases/tag/2.3.6321). Could you please check to see if the issue has been solved now?

**Comment by fmms** [comment]
thanks. it is fiex indeed.
